packages/models/.eslintrc.cjs

```diff
@@ -24,5 +22,23 @@ module.exports = {
         jest: true,
       },
     },
+    {
+      files: ['**/*.js'],
+      rules: {
+        // TODO: fix these and remove this override
+        '@typescript-eslint/adjacent-overload-signatures': 'warn',
+        '@typescript-eslint/ban-ts-comment': 'warn',
+        '@typescript-eslint/explicit-module-boundary-types': 'warn',
+        '@typescript-eslint/no-empty-function': 'warn',
+        '@typescript-eslint/no-this-alias': 'warn',
+        '@typescript-eslint/no-unsafe-argument': 'warn',
+        '@typescript-eslint/no-unsafe-assignment': 'warn',
+        '@typescript-eslint/no-unsafe-call': 'warn',
+        '@typescript-eslint/no-unsafe-member-access': 'warn',
+        '@typescript-eslint/no-unsafe-return': 'warn',
+        '@typescript-eslint/restrict-plus-operands': 'warn',
+        '@typescript-eslint/restrict-template-expressions': 'warn',
```

Comment:
We're currently violating all of these?

Reply:
The legacy javascript code is — we didn't typecheck these files before. That's why I turned these warnings off only for js files.